PublicMapping/publicmapping.github.com
# Create scaffold Jekyll site and set up development environment

This sets up STRTA, a Vagrant and Docker development environment, and a basic Jekyll config along the lines of what we did in https://github.com/azavea/redistricting-the-nation-website/commit/599f75076c583771232f4cff1770acf4385c4b90.

Resolves #2 

## Testing Instructions

- Run `./scripts/setup` to create a development VM.
- Run `vagrant ssh` to get a shell in the VM.
- Run `./scripts/server` to run a Jekyll build and development server. Verify that the site is being served at the link provided in the console output. Currently the site just redirects to http://publicmapping.github.com/districtbuilder/.
- Confirm that Jekyll build output is in the `_site` directory.
